### PR TITLE
add 26 remove-duplicates-from-sorted-array

### PR DESCRIPTION
**Pull request by kobakobashu**
20m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code

File: 26.remove-duplicates-from-sorted-array.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        space: O(1)
+        """
+        if len(nums) == 0:
+            return 0
+        i = 0
+        for j in range(1, len(nums)):
+            if nums[j] != nums[i]:
+                i += 1
+                nums[i] = nums[j]
+        return i+1
```
Comment (ryuji0123) [inline 26.remove-duplicates-from-sorted-array.py:22]: return i + 1

File: 26.remove-duplicates-from-sorted-array.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+class Solution:
+    def removeDuplicates(self, nums: List[int]) -> int:
+        """
+        time: O(len(nums))
+        space: O(1)
+        """
+        if len(nums) == 0:
+            return 0
+        i = 0
```
Comment (ryuji0123) [inline 26.remove-duplicates-from-sorted-array.py:17]: i, j are unclear

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
Please check my code

**Review by ryuji0123 — APPROVED** [review]
LGTM

**Pull request merged**